views/js/runner/plugins/navigation/limitBackButton.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * Plugin that limits the use of the back button only when there is no responses in the current item.
+ * So, once a user has entered a response to any interaction, the back button will be disabled.
+ * We can check if an interaction is already answered by listening to `responsechange` on item runner.
+ *
+ * @author Ricardo Proença <ricardo@taotesting.com>
+ */
+define([
+    "jquery",
+    "lodash",
+    'taoTests/runner/plugin'
+], function ($, _, pluginFactory) {
+    'use strict';
+
+    var pluginName = "limitBackButton";
+
+    var defaultEventName = "change";
+
+    var itemInteractions = {};
```

Comment:
Caution, since this is related to a plugin instance, this should not be a module variable. What will happen if two instances of the same plugins are created?

Reply:
Removed.